openstack/designate/Chart.yaml

```diff
@@ -42,3 +42,8 @@ dependencies:
   - name: linkerd-support
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
     version: 1.1.0
+  - name: redis
+    alias: api-ratelimit-redis
+    repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
+    version: 1.5.3
```

Comment:
There is a new redis chart version 2.0.0 which supports automatic password generation.
Please update accordingly. See example: https://github.com/sapcc/helm-charts/pull/7971/files